### ISSUE TITLE
Add some swarm APIs

### ISSUE THREAD

**Issue by Lycanthrope** [jmlich/foursail#6]
Since the basic implementation of the application many features have changed. Foursquare application was split into two for other OS. Do you have any ideas to add swarm apis which will introduce some new features into that application? e.g. check-in friends, stickers, scoreboard. 
It can be a combination of main features from two applications in one cool app.

**Comment by Maledictus** [comment]
There is no any swarm api available

**Comment by Lycanthrope** [comment]
Yes, you are correct. I have mixed up the name of swarm API with something else, my bad.
My question is about adding 
- stickers to checkins. I saw some kind of documentation [in this repo](https://github.com/DirkSonguer/bb10-swirl/blob/master/_data/Foursquare%20API%20Responses/sticker/stickers_documentation.txt) and an example of usage on [stackoverflow](https://stackoverflow.com/questions/46162821/how-do-i-add-swarm-stickers-to-a-checkin-request).
- scores of checkin from [Get Details of a Check-in](https://developer.foursquare.com/docs/api/checkins/details).
- posibility to check in a friend. Looks like mentions from [Create Check-in](https://developer.foursquare.com/docs/api/checkins/add) should work.
